indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,324 @@
+    {% set break = entry.object %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color"
+    >
+        {{ _render_entry_header(entry, config, program_config) }}
+    </section>
+{% endmacro %}
+
+{% macro _render_contrib(entry, config, program_config, show_children_location) %}
+    {% set contrib = entry.object %}
+    {% set speakers = contrib.speakers %}
+    {% set subcontribs = contrib.subcontributions %}
+    {% set show_contrib_content = subcontribs or (config.show_abstract and contrib.description) %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color {{ 'only-round-top' if show_contrib_content else ''}}"
+    >
+        {{ _render_entry_header(entry, config, program_config) }}
+    </section>
+    {% if show_contrib_content %}
+        <div class="session-block-content">
+            {{ _render_description(contrib.description) }}
+            {% if subcontribs %}
+                {{ _render_subcontribs(contrib, config, show_children_location) }}
+            {% endif %}
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro _render_session_block(entry, config, program_config, show_children_location) %}
+    {% set session_block = entry.object %}
+    {% set sess = session_block.session %}
+    {% set conveners = session_block.person_links %}
+    {% set contributions = session_block.contributions %}
+    {% set show_session_block_content = entry.children or (config.show_session_description and sess.description) %}
+
+    <section
+        id="entry-{{ entry.id }}"
+        class="session-block-title inherited-bg-color inherited-txt-color {{ 'only-round-top' if show_session_block_content else ''}}"
+    >
+        {{ _render_entry_header(entry, config, program_config) }}
+    </section>
+    {% if show_session_block_content %}
+        <div class="session-block-content">
+            {% if config.show_session_description and sess.description %}
+                {{ _render_description(sess.description) }}
+            {% endif %}
+            {% if entry.children %}
+                {{ _render_block_child_entries(sess, entry.children, config, show_children_location) }}
+            {% endif %}
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro _render_block_child_entries(sess, children, config, show_children_location) %}
+    <div class="session-block-content-box inherited-border-color">
+        <ul class="contrib-timeline">
+            {% for child in children %}
+                <li class="inherited-border-color">
+                    <div class="contrib-heading">
+                        {% if not sess.is_poster %}
+                            <span class="timebox inherited-bg-color inherited-txt-color">
+                                {{ format_interval(child.start_dt.astimezone(child.event.tzinfo), child.end_dt.astimezone(child.event.tzinfo), format='HHmm') }}
```

Comment:
Are you sure this should be `format_interval` and not `format_skeleton` (even though the call args like `format` do seem to indicate you want a specific format here)?

`HHmm` looks like a skeleton since I guess you want sth. like `13:00` and not `1300`.

Reply:
It's correct, see: https://github.com/indico/indico/pull/6558#discussion_r1830946645 (something we can fix in a separate PR)

Reply:
Ah, indeed, nevermind then. Let's maybe rename `format` to `skeleton` in our wrapper...